include/h2o.h

```diff
@@ -961,6 +961,14 @@ struct st_h2o_conn_t {
      * callbacks
      */
     const h2o_conn_callbacks_t *callbacks;
+
```

Comment:
```suggestion
```
I do not think there's a reason to add an empty line here, when there's none between the members above?